includes/namespace.php

```diff
@@ -103,6 +103,14 @@ function activate( $network_wide = false ) {
 		);
 	}
 
+	if ( version_compare( get_bloginfo( 'version' ), WEBSTORIES_MINIMUM_WP_VERSION, '<' ) ) {
+		wp_die(
+		/* translators: %s: WordPress version number */
+			esc_html( sprintf( __( 'Web Stories requires WordPress %s or higher.', 'web-stories' ), WEBSTORIES_MINIMUM_WP_VERSION ) ),
```

Comment:
Should there be a link back to admin screen? I had to press the back button in my broswer.

Reply:
Hmm something to consider in new ticket I think, as we'd also have to do it for the PHP check.

IIRC some plugins use an admin notice instead of `wp_die`, and `deactivate_plugin` to instantly deactivate again. That might be an option for us, too.

Reply:
Yeah, just something to think about. I may create a ticket later.